src/processor/operator/scan/scan_rel_table.cpp

```diff
@@ -1,11 +1,73 @@
 #include "processor/operator/scan/scan_rel_table.h"
 
+#include "binder/expression/expression_util.h"
+
 using namespace kuzu::common;
 using namespace kuzu::storage;
 
 namespace kuzu {
 namespace processor {
 
+static std::string relToString(const binder::RelExpression& rel) {
```

Comment:
We might want this in RelExpression so that we can reuse when printing logical operator?